src/RulesTest/Library/FrameworkDoesNotSupportUIAutomationTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Axe.Windows.RulesTest.Library
+{
+    [TestClass]
+
+    public class FrameworkDoesNotSupportUIAutomationTests
+    {
+        private readonly static Rules.IRule Rule = new Rules.Library.FrameworkDoesNotSupportUIAutomation();
+        private Mock<IA11yElement> _elementMock;
+
+        [TestInitialize]
+        public void BeforeEach()
+        {
+            _elementMock = new Mock<IA11yElement>(MockBehavior.Strict);
+        }
+
+        [TestMethod]
+        public void Condition_FrameworkIsNotWin32_ReturnsFalse()
+        {
+            string[] nonWin32Values = { FrameworkId.DirectUI, FrameworkId.Edge, FrameworkId.InternetExplorer, FrameworkId.WinForm, FrameworkId.WPF, FrameworkId.XAML, "NotWin32" };
+
+            foreach (string nonWin32Value in nonWin32Values)
+            {
+                _elementMock.Setup(m => m.Framework)
+                    .Returns(FrameworkId.DirectUI)
```

Comment:
Don't you intend to return `nonWin32Value`?

Reply:
Yes. The first version had a hardcoded value and I forgot to update it when I added the loop